drivers/wifi-client/driver.js

```diff
@@ -576,7 +628,7 @@ class UnifiDriver extends Homey.Driver {
 
                     let name = client.name
                     if (typeof name === 'undefined') name = client.hostname;
-
+                    if (typeof name === 'undefined') name = client.mac;
```

Comment:
You had issues with clients having no hostname ? 😄 (i've not seen this sofar.. :) )